run.sh

```diff
@@ -51,24 +51,25 @@ main() {
   # Download WordPress
   # ------------------
   if [ ! -f /app/wp-settings.php ]; then
-    h3 "Downloading wordpress"
+    h2 "Downloading wordpress"
     chown -R www-data:www-data /app /var/www/html
     WP core download |& loglevel
+    h3 "Wordpress downloaded"
     STATUS "${PIPESTATUS[0]}"
   fi
 
   # Wait for MySQL
   # --------------
   h2 "Waiting for MySQL to initialize..."
+  printf "%b " "${CYAN}${BOLD}  ->${NC} "
   while ! mysqladmin ping --host="$DB_HOST" --password="$DB_PASS" --silent; do
     sleep 1
   done
 
-  h1 "Begin WordPress Configuration"
-
-  h3 "Generating wp.config.php file"
+  h2 "Generating wp-config.php file"
   rm -f /app/wp-config.php
   WP core config |& loglevel
+  h3 "wp-config.php created"
```

Comment:
I'd move the text from the above `h2` to this `h3` block and remove the `h2` block for the same reason as above. This has the potential to pass or fail and the wording here to me sounds like it's always passing.